src/glyphnote.ts

```diff
@@ -1,15 +1,22 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 
+import { Vex } from './vex';
 import { BoundingBox } from './boundingbox';
 import { Glyph } from './glyph';
 import { Note, NoteStruct } from './note';
+import { ModifierContext } from './modifiercontext';
+import { ModifierContextState } from './modifiercontext';
+
 export interface GlyphNoteOptions {
   ignoreTicks?: boolean;
   line?: number;
 }
 
 export class GlyphNote extends Note {
   protected options: GlyphNoteOptions;
+  static format(notes: GlyphNote[], state: ModifierContextState): boolean {
```

Comment:
Do I need to do this?  I am following the pattern of StaveNote, and I needed to do this to get the format() of the modifiers called.  But maybe there is a way I can add the modifiers and not the GlyphNote itself.

Reply:
Indeed I do not, I can just format the modifiers and not the thing itself.  Removed the empty call.